src/Admin.php

```diff
@@ -35,6 +35,9 @@ public static function init() {
     // Exposes SVG images in media library.
     add_filter('wp_prepare_attachment_for_js', __CLASS__ . '::wp_prepare_attachment_for_js');
     add_action('admin_head', __CLASS__ . '::admin_head');
+
+    // Excludes users with subscriber role from the author dropdown select list.
+    add_filter('wp_dropdown_users_args', __CLASS__ . '::wp_dropdown_users_args', 10, 1);
```

Comment:
`1` is the default value for the number of callback arguments, you don't need to specify it.